### PR TITLE
Carrier metadata update for UA - 380 (en)

### PR DESCRIPTION
**Pull request by burunduk**
From: https://www.itu.int/dms_pub/itu-t/oth/02/02/T02020000DB0005PDFE.pdf

### CODE REVIEW

File: resources/carrier/en/380.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright (C) 2013 The Libphonenumber Authors
+# Copyright (C) 2018 The Libphonenumber Authors
```
Comment (kkeshava) [inline resources/carrier/en/380.txt]: Please revert to 2013. It is the initial creation of the file. We do not change the year every time we edit the file :)

File: resources/carrier/en/380.txt [hunk 2]
```diff
@@ -13,19 +13,26 @@
 # limitations under the License.
 
 # Data taken from:
-# https://www.itu.int/dms_pub/itu-t/oth/02/02/T02020000DB0004PDFE.pdf
+# https://www.itu.int/dms_pub/itu-t/oth/02/02/T02020000DB0005PDFE.pdf
```
Comment (kkeshava) [inline resources/carrier/en/380.txt]: Can you change the URL as below. This is dynamic and we can't keep changing it always.

\# Data taken from: https://www.itu.int/oth/T02020000DB/en

File: resources/carrier/en/380.txt [hunk 3]
```diff
@@ -13,19 +13,26 @@
...
-38063|Life
+38063|Lifecell
 38066|Vodafone
 38067|Kyivstar
 38068|Kyivstar
-38073|Life
+38073|Lifecell
+380891|Datagroup
+380892|Ukrtelecom
+380893|T.R. Communicatio
```
Comment (kkeshava) [inline resources/carrier/en/380.txt]: As per http://trcommunication.kiev.ua/ you can make it 'T.R. Communication'

File: resources/carrier/en/380.txt [hunk 4]
```diff
@@ -13,19 +13,26 @@
...
+380894|Atlantic Telecom
+380895|Lincom-300
+380897|Kyivstar
+380899|Velton Telecom
 38091|TriMob
-38092|PEOPLEnet
-38093|Life
+38092|Telesystems
```
Comment (kkeshava) [inline resources/carrier/en/380.txt]: We already have a comment in the file above which says "Telesystems of Ukraine is commonly known as PEOPLEnet", so please revert it.

File: resources/carrier/en/380.txt [hunk 5]
```diff
@@ -13,19 +13,26 @@
...
 38068|Kyivstar
-38073|Life
+38073|Lifecell
+380891|Datagroup
+380892|Ukrtelecom
+380893|T.R. Communicatio
+380894|Atlantic Telecom
+380895|Lincom-300
+380897|Kyivstar
+380899|Velton Telecom
```
Comment (kkeshava) [inline resources/carrier/en/380.txt]: All the prefixes starting with 89 are more of Voip numbers than Mobile. So we do not add carrier details for them. So please remove them

File: resources/carrier/en/380.txt (hunk 1 above)
Comment (burunduk) [reply resources/carrier/en/380.txt]: This  year only creation?

File: resources/carrier/en/380.txt (hunk 2 above)
Comment (burunduk) [reply resources/carrier/en/380.txt]: Fixed.

File: resources/carrier/en/380.txt (hunk 4 above)
Comment (burunduk) [reply resources/carrier/en/380.txt]: Fixed.

File: resources/carrier/en/380.txt (hunk 5 above)
Comment (burunduk) [reply resources/carrier/en/380.txt]: Fixed. But why?

File: resources/carrier/en/380.txt (hunk 1 above)
Comment (kkeshava) [reply resources/carrier/en/380.txt]: No, this file was there before only, it was created back in 2013 for the first time. You are now just making few changes :)

File: resources/carrier/en/380.txt [hunk 6]
```diff
@@ -13,19 +13,19 @@
...
 
 # Data taken from:
-# https://www.itu.int/dms_pub/itu-t/oth/02/02/T02020000DB0004PDFE.pdf
+# https://www.itu.int/oth/T02020000DB/en
 
 # Telesystems of Ukraine is commonly known as PEOPLEnet.
 
 38050|Vodafone
-38063|Life
+38063|Lifecell
```
Comment (kkeshava) [inline resources/carrier/en/380.txt]: One last correction and then we are good to go :)

As per https://www.lifecell.ua/en/about_lifecell/main/ it should be in lower case so can you change 'Lifecell' to 'lifecell'

**Comment by burunduk** [comment]
Oh, correcting :)

**Comment by kkeshava** [comment]
Thank you for contributing to libphonenumber! The changes look good to us :)

This PR is approved now. Please update your branch and once it is done successfully, please click on 'Squash and merge'.

**Pull request closed without merging**